lessonbuilder/tool/pom.xml

```diff
@@ -201,7 +201,11 @@
              <groupId>net.sf.opencsv</groupId>
              <artifactId>opencsv</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>org.jsoup</groupId>
+            <artifactId>jsoup</artifactId>
+            <version>LATEST</version>
```

Comment:
Is there a reason for this? As it leads to unreproducible builds where jsoup releases a new version and all of a sudden Sakai stops building.